database/migrations/2020_12_10_create_school_utilities_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateStudentAdditionalDataTable extends Migration
```

Comment:
rename the class name as  CreateSchoolUtilitiesTable

Reply:
Done